angular.json

```diff
@@ -103,6 +101,20 @@
               ],
               "serviceWorker": true,
               "ngswConfigPath": "ngsw-config.json"
+            },
+            "local": {
```

Comment:
dev is environment.ts, staging environment.stage.ts, and prod. which environment is local

Reply:
yea thats correct. local is for when running the be locally on your own machine

Reply:
yeah thats dev

Reply:
with debug zones enabled

Reply:
dev `environment.ts` is for consuming FE controlled mock api's with `mocky.io` during local development.

I added a new local environment file `environment.local.ts` so provide the ability to consume locally running docker based BE during local development

- so as before to spin up app for development with mocky `npm run start`
- and to spin up app for development with local BE `npm run start:local`